Automattic/wp-calypso
# New G Suite Stats Nudge Copy

#### Changes proposed in this Pull Request

* Change copy in G Suite Stats Nudge to remove references to email the user may already have, avoiding confusion

#### Testing instructions



**Before:** <img width="1050" alt="Screen Shot 2019-09-16 at 3 43 47 PM" src="https://user-images.githubusercontent.com/2810519/64988398-63f35c80-d899-11e9-81d8-9d21d3d67ab1.png">


**After:** <img width="1045" alt="Screen Shot 2019-09-16 at 3 43 16 PM" src="https://user-images.githubusercontent.com/2810519/64988367-4f16c900-d899-11e9-81f0-8a2c7eeff7dc.png">

* Navigate to `/stats/day/` for a blog with a domain
* Observe the new G Suite Nudge copy and that it matches the "Before" screenshot above.
* **Note:** You may have dismissed this banner before, in which case it will not appear. To un-dismiss it follow the instructions in this screenshot: <img width="1057" alt="Screen Shot 2019-09-16 at 3 41 29 PM" src="https://user-images.githubusercontent.com/2810519/64988258-14ad2c00-d899-11e9-93ae-bf41c2bd07f8.png">

Fixes #32409
